yaptide/routes/utils/tokens.py

```diff
@@ -9,6 +9,7 @@
 _Refresh_Token_Expiration_Time = 120  # minutes
 _Access_Token_Expiration_Time = 10  # minutes
 _Keycloak_Token_Expiration_Time = 30  # minutes
+_Simulation_Token_Expiration_time = 7  # days
```

Comment:
I know there is a comment here mentioning the unit of time. 
All variables above are in minutes, you introduce new one in days.
I'd suggest to be more constistent and follow one of the options
- define all the variables as `timedelta` object: `_Simulation_Token_Expiration_time_days = timedelta(days=7)`
- add suffix to variable which identifies unit of time, like `_Simulation_Token_Expiration_time_days = 7`
- move all the variables to minutes

Personally I am in favour of using timedelta objects here. When you use them its easy to convert them to minutes, seconds of whatever

Reply:
It's a change made by original author of these changes so i didn't really noticed but you right about those days.